spec-main/api-browser-window-spec.ts

```diff
@@ -58,6 +52,28 @@ describe('BrowserWindow module', () => {
     })
   })
 
+  describe('garbage collection', () => {
+    const v8Util = process.electronBinding('v8_util')
+    afterEach(closeAllWindows)
+
+    it('window does not get garbage collected when opened', (done) => {
+      const w = new BrowserWindow({ show: false })
+      // Keep a weak reference to the window.
+      const map = v8Util.createIDWeakMap<Electron.BrowserWindow>()
```

Comment:
The window removes itself from the `BrowserWindow.getAllWindows` map when it is closed, so this API can not be used to test if a window is garbage collected. And this behavior is intended otherwise ghost window would be return in the API.

Weak map is the only tool we can use in existing APIs.